tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -630,6 +631,7 @@ unlink_files(void)
 		rc = unlink(file);
 		if ((rc == -1) && (errno != ENOENT)) {
 			ERROR("Error %d: unlink(%s)\n", errno, file);
+			free(file);
```

Comment:
@behlendorf  
I see file is allocated by malloc above. as below
"
 	file = malloc(PATH_MAX);
 	if (file == NULL) {
 		rc = ENOMEM;
 		ERROR("Error %d: malloc(%d) bytes for file name\n",
 		    rc, PATH_MAX);
 		goto out;
 	}"